apps/authz/kinds/managedpermission.cue

```diff
@@ -0,0 +1,24 @@
+package kinds
+
+import (
+	"github.com/grafana/grafana/apps/authz/kinds/v0alpha1"
+)
+
+managedpermission: {
+	kind:       "ManagedPermission"
```

Comment:
Can we use another name, ie `ResourcePermission`? Managed seems like makes sense only in RBAC context, this is implementation details.

Reply:
Yes, I meant to change this, thanks for spotting :)